.github/workflows/generate-phapi-synthetic-data-spreadsheet.yaml

```diff
@@ -44,20 +44,35 @@ jobs:
           ref: qa
           fetch-depth: 0
           path: synthetic-records
-      - name: Generate Spreadsheet
+      - name: Generate PHAPI Spreadsheet
         env:
           GITHUB_TOKEN_USERNAME: ${{ secrets.BUILD_GITHUB_USERNAME }}
           GITHUB_TOKEN: ${{ secrets.BUILD_GITHUB_TOKEN }}
         run: |
           export LIGHTHOUSE_SETTINGS=${GITHUB_WORKSPACE}/consumer-docs/.github/build/settings.xml
-          echo "Building spreadsheet..."
+          echo "Building PHAPI spreadsheet..."
```

Comment:
I think we should explicitly call out resources here. The addition of the properties keeps us backwards compatible, but moving forward, we should not bank on those defaults for the PHAPI generation.